hraven-etl/src/main/java/com/twitter/hraven/mapreduce/GraphiteOutputFormat.java

```diff
@@ -163,6 +167,50 @@ public void write(EnumWritable<HravenService> serviceKey, HravenRecord value) th
           throw new IOException(e);
         }  
       }
+        } else {
+            String multipleConfigInput = context.getConfiguration().get("hraven.multiple.config.files");
+            String[] multipleConfigs = multipleConfigInput.split(",");
```

Comment:
load this config only once (not per metric) - and store statically in this class (Map<configFile, graphitesinkconfig>) objects)